images/doppler-kubernetes-operator/README.md

```diff
@@ -0,0 +1,29 @@
+<!--monopod:start-->
+# images/doppler-kubernetes-operator
+| | |
+| - | - |
+| **OCI Reference** | `cgr.dev/chainguard/images/doppler-kubernetes-operator` |
+
+
+* [View Image in Chainguard Academy](https://edu.chainguard.dev/chainguard/chainguard-images/reference/images/doppler-kubernetes-operator/overview/)
+* [View Image Catalog](https://console.enforce.dev/images/catalog) for a full list of available tags.
+* [Contact Chainguard](https://www.chainguard.dev/chainguard-images) for enterprise support, SLAs, and access to older tags.*
+
+---
+<!--monopod:end-->
+
+<!--overview:start-->
+Automatically sync secrets from Doppler to Kubernetes and auto-reload deployments when secrets change.
+<!--overview:end-->
+
+<!--getting:start-->
+## Download this Image
+The image is available on `cgr.dev`:
+
+```
+docker pull cgr.dev/chainguard/images/doppler-kubernetes-operator:latest
+```
+<!--getting:end-->
+
+<!--body:start-->
```

Comment:
Thanks for putting this together. We'll need some examples here on how to deploy the image. i.e if its docker and/or helm.

From the test it looks like via an operator, so some notes here on how to install / run the operator, and override the image parameters would be great

Reply:
Add it to the TESTING.md file